### ISSUE TITLE
NATS datastore

### ISSUE THREAD

**Issue by rmarku** [k3s-io/docs#164]
There is no info on how to use NATS datastore with k3s in https://docs.k3s.io/datastore

**Comment by brandond** [comment]
It would be great if the nats team would contribute some docs. They contributed the support to kine, and that has been pulled into k3s, but we don't have any good guidance on how exactly they recommend using it.

**Comment by brandond** [comment]
cc @bruth @matthewdevenny @jnmoyne

**Comment by bruth** [comment]
Absolutely. Assigning myself. Should have something this week. In the meantime, a basic setup would look like this:

```
k3s server --datastore-endpoint=nats://localhost:4222
```

The current set of endpoint parameters are listed here: https://github.com/k3s-io/kine/blob/master/examples/nats.md. Do note that the current release of k3s does not have the embedded NATS support yet (requires a build tag).
